cinema/serializers.py

```diff
@@ -125,9 +125,9 @@
     def validate(self, attrs):
         data = super(TicketSerializer, self).validate(attrs=attrs)
         Ticket.validate_ticket(
-            attrs["row"], 
-            attrs["seat"], 
-            attrs["movie_session"].cinema_hall, 
+            attrs["row"],
+            attrs["seat"],
+            attrs["movie_session"].cinema_hall,
             ValidationError
```

Comment:
Before calling `Ticket.validate_ticket`, check if `attrs['movie_session']` is not `None`. This will prevent errors if the `movie_session` attribute is missing from `attrs`.